src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -84,6 +84,11 @@ def input_values(self):
     def output_columns(self):
         pass
 
+    @property
+    @abstractmethod
+    def output_name(self) -> str:
```

Comment:
Funny b/c I'm pretty sure this was the original implementation we proposed. Can't recall why this got axed